wagtail/core/models.py

```diff
@@ -1930,6 +1930,147 @@ def copy(self, recursive=False, to=None, update_attrs=None, copy_revisions=True,
+
+        update_attrs = {
+            'alias_of': self,
+
+            # Aliases don't have revisions so the draft title should always match the live title
+            'draft_title': self.title,
+
+            # Likewise, an alias page can't have unpublished changes if it's live
+            'has_unpublished_changes': not self.live,
+        }
+
+        if update_slug:
+            update_attrs['slug'] = update_slug
+
+        if update_locale:
+            update_attrs['locale'] = update_locale
+
+        if user:
+            update_attrs['owner'] = user
+
+        # When we're not copying for translation, we should give the translation_key a new value
+        if reset_translation_key:
+            update_attrs['translation_key'] = uuid.uuid4()
+
+        alias, child_object_map = _copy(specific_self, update_attrs=update_attrs, exclude_fields=exclude_fields)
+
+        # Update any translatable child objects
+        for (child_relation, old_pk), child_object in child_object_map.items():
+            if isinstance(child_object, TranslatableMixin):
+                if update_locale:
+                    child_object.locale = update_locale
+
+                # When we're not copying for translation, we should give the translation_key a new value for each child object as well
+                if reset_translation_key:
+                    child_object.translation_key = uuid.uuid4()
+
+        # Save the new page
+        if _mpnode_attrs:
+            # We've got a tree position already reserved. Perform a quick save
+            alias.path = _mpnode_attrs[0]
+            alias.depth = _mpnode_attrs[1]
+            alias.save(clean=False)
+
+        else:
+            if parent:
+                if recursive and (parent == self or parent.is_descendant_of(self)):
+                    raise Exception("You cannot copy a tree branch recursively into itself")
+                alias = parent.add_child(instance=alias)
+            else:
+                alias = self.add_sibling(instance=alias)
+
+            _mpnode_attrs = (alias.path, alias.depth)
+
+        _copy_m2m_relations(specific_self, alias, exclude_fields=exclude_fields)
+
+        # Log
+        if log_action:
+            source_parent = specific_self.get_parent()
+            PageLogEntry.objects.log_action(
+                instance=alias,
+                action=log_action,
+                user=user,
+                data={
+                    'page': {
+                        'id': alias.id,
+                        'title': alias.get_admin_display_title()
+                    },
+                    'source': {'id': source_parent.id, 'title': source_parent.get_admin_display_title()} if source_parent else None,
+                    'destination': {'id': parent.id, 'title': parent.get_admin_display_title()} if parent else None,
+                },
+            )
+            if alias.live:
+                # Log the publish if the use chose to keep the copied page live
```

Comment:
"if the use chose to keep the copied page live" can be deleted here, as there is no `keep_live` option.